Lib/idlelib/idle_test/test_searchbase.py

```diff
@@ -5,7 +5,7 @@
 '''
 import unittest
 from test.support import requires
-from tkinter import Tk, Toplevel, Frame ##, BooleanVar, StringVar
+from tkinter import Tk, Frame  ##, BooleanVar, StringVar
```

Comment:
They are used in commented code a few lines below. I don't want to remove this commented code, since I don't know this file.